<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStatistics</class>
 <widget class="QDialog" name="DialogStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(134,237,231);</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>21</y>
     <width>333</width>
     <height>252</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="dataManaButton">
        <property name="minimumSize">
         <size>
          <width>111</width>
          <height>101</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgb(134,237,231); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:focus { background-color: rgb(244,236,2); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pingbao.qrc">
          <normaloff>:/new/prefix1/ICON/6/1 (41).png</normaloff>:/new/prefix1/ICON/6/1 (41).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>111</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;数据统计&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;管理&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="weiyiCButton">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>101</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgb(134,237,231); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:focus { background-color: rgb(244,236,2); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pingbao.qrc">
          <normaloff>:/new/prefix1/ICON/6/1 (42).png</normaloff>:/new/prefix1/ICON/6/1 (42).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>141</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;位移传感器&lt;/p&gt;&lt;p align=&quot;center&quot;&gt; 数据统计&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="yaliCButton">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>101</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgb(134,237,231); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:focus { background-color: rgb(244,236,2); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pingbao.qrc">
          <normaloff>:/new/prefix1/ICON/6/1 (43).png</normaloff>:/new/prefix1/ICON/6/1 (43).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>111</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;压力传感器&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;数据统计&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pingbao.qrc"/>
 </resources>
 <connections/>
</ui>
